### DISCUSSION TITLE
Tree Orgchart

### DISCUSSION THREAD

**Discussion by jurivero** [pnp/sp-dev-fx-webparts#1779] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/13647324/111979747-d15dfa00-8adb-11eb-9e2c-14e3f4912a0e.png)

I am getting the following error
Uncaught (in promise) InteractionRequiredAuthError: AADSTS65001: The user or administrator has not consented to use the application with ID 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxx' named 'SharePoint Online Client Extensibility Web Application Principal'. Send an interactive authorization request for this user and resource.

I check the azure portal and in the app registration I see on 
SharePoint Online Client Extensibility Web Application Principal API Permissions
![image](https://user-images.githubusercontent.com/13647324/111979385-64e2fb00-8adb-11eb-9311-c4196a3687e0.png)

I do not know where else I could check when I have the permission to read microsoft graphs.

**Comment by AriGunawan** [comment, marked as answer]
Hi @jurivero ,
Have you approved the related API requests on your SharePoint admin center page?
You can find all pending API requests here: [https://{your-tenant}-admin.sharepoint.com/_layouts/15/online/AdminHome.aspx#/webApiPermissionManagement ](https://{your-tenant}-admin.sharepoint.com/_layouts/15/online/AdminHome.aspx#/webApiPermissionManagement)

[Related documentation](https://docs.microsoft.com/en-us/sharepoint/api-access).
